src/algorithms/standard/minmax.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#ifndef ESSENTIA_MINMAX_H
+#define ESSENTIA_MINMAX_H
+
+#include "algorithm.h"
+
+namespace essentia {
+namespace standard {
+
+class MinMax : public Algorithm {
```

Comment:
Does it make sense to have a single algorithm with both behaviours, or do we prefer 2 separate algorithms?

Reply:
I think it's fine to have both here.